app/views/schedule/confirmation.html.erb

```diff
@@ -1,30 +1,30 @@
 <% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
 <% content_for :title, t("schedule.confirmation.title") %>
-
-<% scheduled_time = @edition.scheduled_publishing_datetime.strftime("%l:%M%P").strip %>
-<% scheduled_date = @edition.scheduled_publishing_datetime.strftime("%-d %B %Y") %>
+<% datetime = @edition.scheduled_publishing_datetime %>
+<% time = datetime&.strftime("%l:%M%P")&.strip %>
+<% date = datetime&.strftime("%-d %B %Y") %>
```

Comment:
as we have these a few times it could make sense to have scheduled_date and scheduled_time helper methods

Reply:
Now in the Scheduled Datetime Helper.